engine/message/db.go

```diff
@@ -575,6 +594,49 @@ func (db *DB) _SendMessages(ctx context.Context, send SendFunc, status StatusFun
 		return errors.Wrap(err, "clear disabled status updates")
 	}
 
+	// if twilio is disable, create an entry to notify the user
+	cfg := config.FromContext(ctx)
+	if !cfg.Twilio.Enable {
+		rows, err := tx.Stmt(db.twilioDisabled).QueryContext(execCtx)
```

Comment:
Use `StmtContext` for new code.

```suggestion
		rows, err := tx.StmtContext(ctx, db.twilioDisabled).QueryContext(execCtx)
```